lib/senkyoshi/models/root_resource.rb

```diff
@@ -0,0 +1,23 @@
+require "senkyoshi/models/resource"
+
+module Senkyoshi
+  ##
+  # Class to represent a resource constructed from a single 'dat' file.
+  ##
+  class RootResource < Resource
```

Comment:
I wonder if the names should be switched because if the RootResource is inheriting from Resource and it feels like it should be the other way around.